packages/vuetify/src/components/VAlert/VAlert.sass

```diff
@@ -144,3 +144,22 @@
     text-transform: $alert-title-text-transform
     word-break: $alert-title-word-break
     word-wrap: $alert-title-word-wrap
+
+@media (forced-colors: active)
+  .v-alert
+    border-style: solid
+
+    &--variant-plain
+      border-style: dashed
+
+    &--variant-text,
+    &--variant-plain
```

Comment:
Apart from being duplicated here, `*-plain` primary purpose is to be unstyled. Maybe we should leave it (and `*-text`) without the border (similar to VCard)?

Reply:
Removed border from text & plain variants. 
<img width="2955" height="975" alt="image" src="https://github.com/user-attachments/assets/5e1c9ea1-0a19-4cac-90a5-d2e15fb37dbd" />